daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -70,15 +70,178 @@ spec:
+
+#### High Cardinality without path matching
+
+Configuration:
+```yaml
+http:
+  increasedCardinality: true
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/1",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/2",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/3",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/4",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders/5",status="200"} 1
+```
+
+For each request, a new metric is created with the request path. This process continues for every request made to a new order ID, resulting in unbounded cardinality since the IDs are ever-growing.
+
+
+### HTTP metrics exclude verbs
+
+The `excludeVerbs` option allows you to exclude specific HTTP verbs from being reported in the metrics. This can be useful in high-performance applications where memory savings are critical.
+
+### Examples of excluding HTTP verbs in metrics
+
+Here are some examples demonstrating how to exclude HTTP verbs in Dapr for managing HTTP metrics.
+
+#### Default - Include HTTP verbs
+
+Configuration:
+```yaml
+http:
+  excludeVerbs: false
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="GET",path="/orders",status="200"} 1
+dapr_http_server_request_count{app_id="order-service",method="POST",path="/orders",status="200"} 1
+```
+
+In this example, the HTTP method is included in the metrics, resulting in a separate metric for each request to the `/orders` endpoint.
+
+#### Exclude HTTP verbs
+
+Configuration:
+```yaml
+http:
+  excludeVerbs: true
+```
+
+Metrics generated:
+```
+dapr_http_server_request_count{app_id="order-service",method="",path="/orders",status="200"} 2
+```
+
+In this example, the HTTP method is excluded from the metrics, resulting in a single metric for all requests to the `/orders` endpoint.
+
+### Configuring Custom Latency Buckets
+
+By default, Dapr employs predefined latency buckets for histogram metrics, listed below in milliseconds:
+
+```
+1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1_000, 2_000, 5_000, 10_000, 20_000, 50_000, 100_000
+```
+
+These buckets provide detailed granularity for latency metrics but can result in high cardinality for certain values,
+where a simpler estimate might be adequate.
+
+To tailor the latency buckets according to your specific needs, modify the `spec.metrics.latencyDistributionBuckets`
+field within the Dapr Configuration resource that pertains to your application(s).
+
+The following example shows how to set custom latency buckets:
```

Comment:
Describe what this example is showing. Is this saying that I have now reduced the number of latency buckets records metrics from the default values above ^^?